frontend/www/js/omegaup/components/group/Members.vue

```diff
@@ -55,14 +56,26 @@
           <td>{{ identity.state }}</td>
           <td>{{ identity.school }}</td>
           <td>
-            <a class="glyphicon glyphicon-lock"
+            <a class="glyphicon glyphicon-edit"
+                href="#"
+                v-bind:title="T.groupEditMembersEdit"
+                v-on:click="onEdit(identity)"></a> <a class="glyphicon glyphicon-lock"
                 href="#"
                 v-bind:title="T.groupEditMembersChangePassword"
-                v-on:click="onChangePass(identity.username)"></a>
+                v-on:click="onChangePass(identity.username)"></a> <a class=
+                "glyphicon glyphicon-remove"
+                href="#"
+                v-bind:title="T.groupEditMembersRemove"
+                v-on:click="onRemove(identity.username)"></a>
           </td>
         </tr>
       </tbody>
-    </table><!-- Modal Change Password-->
+    </table><omegaup-identity-edit v-bind:countries="countries"
+         v-bind:identity="identity"
+         v-bind:selectedcountry="identity.country_id"
```

Comment:
nit:

```suggestion
         v-bind:selected-country="identity.country_id"
```

así la propiedad se pasa en camelCase como `selectedCountry`.

Reply:
Listo